tidb-scheduling.md

```diff
@@ -77,6 +77,16 @@ Scheduling is based on information collection. In short, the PD scheduling compo
     * Whether the store is overloaded
     * Labels (See [Perception of Topology](/schedule-replicas-by-topology-labels.md))
 
+You can use PD control to check the status of a TiKV store, which can be Up, Disconnect, Offline, Down, or Tombstone. The following is a description of all statuses and their relationship.
+
++ **Up**: The current TiKV store is in service.
++ **Disconnect**: Heartbeat messages between the PD and the TiKV store are lost for more than 20 seconds. If the lost period exceeds the time specified by `max-store-down-time`, the status changes to "Down".
++ **Down**: Heartbeat messages between the PD and the TiKV store are lost for a time longer than `max-store-down-time` (30 minutes by default). In this status, the TiKV store starts replenishing replicas of each Region on the surviving store.
++ **Offline**: A TiKV store is manually taken offline through PD Control. This is only an intermediate status for the store to go offline. The store in this status moves all its Regions to other "Up" stores that meet the relocation conditions. When `leader_count` and `region_count` (obtained through PD Control) both show `0`, the store status changes to "Tombstone" from "Offline". In the "Offline" status, **do not** disable the store service or the physical server where the store is located. During the process that the store goes offline, if the cluster does not have other target stores that meet the relocation conditions (for example, there are not enough stores that can meet the requirements for the number of replicas in the cluster anymore), the store always is in the "Offline" status.
```

Comment:
@JmPotato @shichun-0415 all comments are addressed, PTAL, thank you!

Reply:
if the cluster does not have other target stores that meet the relocation conditions (for example, there are not enough stores that can meet the requirements for the number of replicas in the cluster anymore), the store always is in the "Offline" status.---**Seems this sentence is still the original one.**